src/screens/QuestionScreen.tsx

```diff
@@ -79,90 +80,108 @@ function getElement(questionOption: QuestionOption) {
+          <OpenTextArea
+            placeholder={questionOption.extra_data.placeholder}
+            value={questionOption.answer?.values?.[0] ?? ''}
+            onChangeText={(value: string) => {
+              questionOption.answer = {
+                id: questionOption.answer?.id ?? 1,
+                values: [value],
+              } as Answer;
+            }}
+          />
+        </>
       );
     case QuestionOptionType.IMAGE:
       return (
-        <ImageSelector
-          value={getMediaURI(questionOption)}
-          onImageSelected={(image: FileSelectedData | null) => {
-            if (image) {
-              questionOption.answer = {
-                id: answerId,
-                values: [image],
-              } as Answer;
-            } else {
-              questionOption.answer = undefined;
-            }
-          }}
-        />
+        <>
+          <QuestionTitle text={'Foto Vraag'}></QuestionTitle>
+          <ImageSelector
+            value={getMediaURI(questionOption)}
+            onImageSelected={(image: FileSelectedData | null) => {
+              if (image) {
+                questionOption.answer = {
+                  id: answerId,
+                  values: [image],
+                } as Answer;
+              } else {
+                questionOption.answer = undefined;
+              }
+            }}
+          />
+        </>
       );
     case QuestionOptionType.VIDEO:
       return (
-        <VideoSelector
-          value={getMediaURI(questionOption)}
-          onVideoSelected={function (videoPath: FileSelectedData | undefined): void {
-            if (videoPath) {
-              questionOption.answer = {
-                id: answerId,
-                values: [videoPath],
-              } as Answer;
-            } else {
-              questionOption.answer = undefined;
-            }
-          }}
-        />
+        <>
+          <QuestionTitle text={'Video vraag'}></QuestionTitle>
+          <VideoSelector
+            value={getMediaURI(questionOption)}
+            onVideoSelected={function (videoPath: FileSelectedData | undefined): void {
+              if (videoPath) {
+                questionOption.answer = {
+                  id: answerId,
+                  values: [videoPath],
+                } as Answer;
+              } else {
+                questionOption.answer = undefined;
+              }
+            }}
+          />
+        </>
       );
     case QuestionOptionType.VOICE:
       return (
-        <AudioRecorder
-          value={getMediaURI(questionOption)}
-          onAudioRecorded={(audio: FileSelectedData | null) => {
-            if (audio) {
-              questionOption.answer = {
-                id: answerId,
-                values: [audio],
-              } as Answer;
-            } else {
-              questionOption.answer = undefined;
-            }
-          }}
-        />
+        <>
+          <QuestionTitle text={'Audio vraag'}></QuestionTitle>
```

Comment:
```suggestion
          <QuestionTitle text={'Audio'}></QuestionTitle>
```

En accessibility label met "Audio opname maken"